run.sh

```diff
@@ -9,36 +9,31 @@ export TOOLCHAIN_ID="1cfc3bd9-3891-4a9d-8503-e1cb10333e38"
 
 # API Key of the user
 export IBM_CLOUD_API_KEY="5nolj6lyv65L6dEZbuAzK5sGLgWmw9VXCFO7xqwoUCwI"
-
-# This is the name of the application that shows up on the Dashboard
-export LOGICAL_APP_NAME="Weather Application"
-
-# This value is prefixed to the build number
-export BUILD_PREFIX=master
-
-# A unique number identifying the build
-export BUILD_NUMBER=1
+export MY_APP_NAME="Weather Application"
+export MY_BUILD_NUMBER="master:1"
```

Comment:
Let have comment above each line explaining what this variable is for, just as we had earlier